cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -92,22 +92,23 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to find out which 2011 data and MC sets exist, and how to get a feel for their content."
 
-- Go to 2011 CMS primary datasets or 2011 CMS simulated datasets,
-- choose a dataset, and read the comments.
+- Go to [2011 CMS primary datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Collision&experiment=CMS&year=2011) or [2011 CMS simulated datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Simulated&experiment=CMS&year=2011),
+- choose a dataset, and read the comments, read also about [simulated dataset names](http://opendata.cern.ch/about/CMS-Simulated-Dataset-Names) FIXME link
```

Comment:
`- choose a dataset, read the comments, and also read about [simulated dataset names](/about/CMS-Simulated-Dataset-Names) FIXME link`